README.md

```diff
@@ -162,21 +168,51 @@ Start rails server:
 bin/rails server
 ```
 
-A simple test that it's working:
-```sh
+Try this simple test, to ensure it's working:
+
+```
 $ curl http://127.0.0.1:3000/healthcheck
 {"checks":{"database":true}}
 ```
+## Generation of API documentation using Rswag
+
+See [Rswag readme](https://github.com/rswag/rswag/blob/master/README.md) for initial setup and/or modifications.
+
+The `swagger` folder in the root directory has one `swagger.yaml` within a version number folder - e.g. `swagger/v4/swagger.yaml`. This file is what defines the swagger ui page displayed at `/api-docs`. This file is generated using rswag's rake task - `rake rswag:specs:swaggerize`.
+
+The `swagger.yaml` file that is generated is defined by a combination of "global" settings in `spec/swagger_helper.rb` and indivual spec files that are, by our convention, stored in `spec/requests/swagger_docs/<version>/*.spec.rb`.
+
+You can generate a new endpoint spec file using:
+```sh
+rails generate rspec:swagger MyController
+```
+
+You can update an existing endpoint by modifying it's spec and then running:
+```sh
+rake rswag:specs:swaggerize
+```
+
+## Threshold configuration files
+
+Files holding details of all thresholds values used in calculating eligibility are stored in `config/thresholds`.
+The file `values.yml` details the start dates for each set of thresholds, and the name of the file from which they should be read.
+
+If a file has the key `test_only` with a value of true, then that file will only be read if the
+`USE_TEST_THRESHOLD_DATA` environment variable is set to true.  This is the default for staging and UAT, and it is
+false for production.
```

Comment:
This seems to be a redundant feature now - as we can put future thresholds into the YAML and access them by passing appropriate dates. The code is still present in the codebase though

Reply:
I've added a note saying it is deprecated functionality. It can be a separate task to delete the code and the bit from the readme together.